### ISSUE TITLE
%expiration% tag incorrectly shows current date in Active Subscription email

### ISSUE THREAD

**Issue by nosegraze** [restrictcontentpro/restrict-content-pro#775]
I've only tested this with Stripe so I'm not sure if it's an issue with other gateways. But here's what happens:

- New person makes payment.
- Expiration obviously doesn't exist at this point.
- When payment is received, the status is set to active. ( https://github.com/restrictcontentpro/restrict-content-pro/blob/master/includes/gateways/class-rcp-payment-gateway-stripe.php#L333 )
- This transition triggers the "Active Subscription" email.
- The %expiration% tag gets the expiration date, which is still empty at this point, so the code ends up turning it into today's date.
- Then further down the expiration date is set. ( https://github.com/restrictcontentpro/restrict-content-pro/blob/master/includes/gateways/class-rcp-payment-gateway-stripe.php#L344 )

In short: the user's status is set to active *before* the expiration date is set, which makes the %expiration% tag inaccurate in Active Subscription emails.

**Comment by pippinsplugins** [comment]
Saw this happen on pippinsplugins.com today.

**Comment by mindctrl** [comment]
@nosegraze want to test `issue/775`?

**Comment by mindctrl** [comment]
Merged.
